opencog/python/README.md

```diff
@@ -1,19 +1,14 @@
 Using Python with the AtomSpace
 ===============================
 
-Add these lines to the end of your ~/.bashrc file:
-```
-PYTHONPATH="$PYTHONPATH:/usr/local/share/opencog/python
-export PYTHONPATH
-```
+#### Installation and Usage
 
-Install the required dependencies:
-```
-cd ~/opencog/opencog/python
-sudo pip install -r requirements.txt --upgrade
```

Comment:
Never mind.  I just looked at it. The file is mis-named, they're not "requirements" they are "prerequisites".